draft-ietf-quic-tls.md

```diff
@@ -1168,89 +1168,232 @@ anticipation of receiving a ClientHello.
 
 # Key Update
 
-Once the handshake is confirmed, it is possible to update the keys. The
-KEY_PHASE bit in the short header is used to indicate whether key updates
-have occurred. The KEY_PHASE bit is initially set to 0 and then inverted
-with each key update.
+Once the 1-RTT keys are established and confirmed, it is possible to update the
```

Comment:
The `handshake is confirmed` has a meaning [formally defined in section 4.1.2](https://quicwg.org/base-drafts/draft-ietf-quic-tls.html#handshake-confirmed). Do we have a formal definition for confirming 1-RTT keys?

Reply:
I think that this is fine.  The reason we needed that definition is that we were hanging a lot from it.  This should however be referencing that definition, because it's handshake confirmation that I intended to use (but failed to).